README.md

```diff
@@ -52,23 +52,29 @@ import (
 var VERSION string
 
 func main() {
-      // Create a new CLI app. This will return a urfave/cli App with some
-      // common initialization.
-      app := entrypoint.NewApp()
-    
-      app.Name = "my-app"
-      app.Author = "Gruntwork <www.gruntwork.io>"
-      
-      // Set the version number from your app from the VERSION variable that is passed in at build time
-      app.Version = VERSION
-      
-      app.Action = func(cliContext *cli.Context) error { 
+    // Create a new CLI app. This will return a urfave/cli App with some
+    // common initialization.
+    app := entrypoint.NewApp()
+
+    app.Name = "my-app"
+	app.Authors = []*cli.Author{
+		{
+			Name:  "Yoriyasu Yano",
+			Email: "yorinasub17@gmail.com",
```

Comment:
Did you mean to put your contact details here?

Reply:
Nope that is accidental! Thanks for flagging. Will delete this comment as well so it isn't highlighted.